src/braket/pennylane_plugin/ops.py

```diff
@@ -262,44 +204,6 @@ def _matrix(cls, *params):
         return np.diag(np.array([1.0, 1.0, np.exp(1.0j * params[0]), 1.0], dtype=complex))
 
 
-class ISWAP(Operation):

```

Comment:
Available now in [PennyLane core](https://github.com/PennyLaneAI/pennylane/blob/ca227436391adf10df1f946cc727a9abb36673eb/pennylane/ops/qubit.py#L545).